src/Core/Enums/SupportedDatabaseProviders.cs

```diff
@@ -0,0 +1,11 @@
+namespace Bit.Core.Enums
+{
+    public enum SupportedDatabaseProviders 
+    {
+        SqlServer,
+        MySql,
+        Postgres,
+        Sqlite,
```

Comment:
😍

Reply:
Removed this option, as it wasn't working